app/controllers/application_controller.rb

```diff
@@ -55,7 +55,7 @@ def clear_saved_form(form_id)
 
   # returns a Bad Request if the incoming host header is unsafe.
   def block_unknown_hosts
-    return if controller_name == 'example'
+    return if controller_name.in?(%w[example okta])
```

Comment:
workaround while this is still coming via localtunnel. Will remove prior to merging and once Kong is set up